### ISSUE TITLE
Editing existing checkbox field throws warning

### ISSUE THREAD

**Issue by dasistwas** [dasistwas/moodle-mod_datalynx#98]
How to reproduce
1) Add field of type checbox
2) Click on "Save and continue" 

Another possibility to reproduce:
Edit an existing field of typt checkbox

`

( ! )  Notice: Array to string conversion in  /var/moodledata31_workspace/localcache/mustache/1581116306/boost/__Mustache_36b0a92f643ddf5f0e38359f27876272.php  on line 80
--

1 | 0.0001 | 398232 | {main}(  ) | .../field_edit.php:0
2 | 0.1436 | 10371104 | datalynxfield_checkbox_form->display(  ) | .../field_edit.php:112
3 | 0.1443 | 10395816 | MoodleQuickForm->display(  ) | .../formslib.php:964
4 | 0.1443 | 10395816 | MoodleQuickForm->toHtml(  ) | .../Common.php:445
5 | 0.1443 | 10395816 | MoodleQuickForm->accept(  ) | .../QuickForm.php:1696
6 | 0.1445 | 10398312 | MoodleQuickForm->accept(  ) | .../formslib.php:1802
7 | 0.1466 | 10445576 | MoodleQuickForm_group->accept(  ) | .../QuickForm.php:1653
8 | 0.1466 | 10445600 | MoodleQuickForm_Renderer->startGroup(  ) | .../group.php:232
9 | 0.1466 | 10446056 | theme_boost\output\core_renderer->mform_element(  ) | .../formslib.php:2946
10 | 0.1466 | 10446104 | MoodleQuickForm_group->export_for_template(  ) | .../outputrenderers.php:4445
11 | 0.1476 | 10493864 | theme_boost\output\core_renderer->mform_element(  ) | .../group.php:182
12 | 0.1476 | 10495704 | theme_boost\output\core_renderer->render_from_template(  ) | .../outputrenderers.php:4488
13 | 0.1477 | 10568752 | __Mustache_36b0a92f643ddf5f0e38359f27876272->render(  ) | .../outputrenderers.php:191
14 | 0.1477 | 10569232 | __Mustache_36b0a92f643ddf5f0e38359f27876272->renderInternal(  ) | .../Template.php:67

`

**Comment by michaelpollak** [comment]
Damn, I'm sorry. 
Caused when I added the attribute size for html validation https://github.com/michaelpollak/moodle-mod_datalynx/commit/34e4fe0fdba331fb4898e1ac5c0a0550fd928bdc#diff-3756bbc057ad30b4df6a68af2a295cd3. 
This code worked everywhere else so I wasn't checking, totally my fault.
Fixed in the current push, will go through this one more time and check all other "size" changes. Thanks for catching this David!
